### ISSUE TITLE
Delete - Refills Vitals

### ISSUE THREAD

**Issue by sunbeam906** [Kyle873/SSR-KMod#2]
Using latest version of this ([7c2267a](https://github.com/Kyle873/SSR-KMod/commit/7c2267acc32a81434efcfe31ca68bef662f25eed)). Wanted to test it and got this:

![image](https://github.com/Kyle873/SSR-KMod/assets/148178/80967be1-68c5-42d9-8ed5-0b0cdc30a034)

Am using 2.5.2 of ue4ss, Standard version which gets injected after game load. The game version is the latest on Steam.

EDIT 1: If Ctrl+R is supposed to log text to console, then that doesn't happen. I figured it might've been cuz I didn't press those keys.

EDIT 2: Found an issue, you forgot a ":" here

https://github.com/Kyle873/SSR-KMod/blob/7c2267acc32a81434efcfe31ca68bef662f25eed/Scripts/commands.lua#L91

However, that still doesn't fix it.

EDIT 3: Also this happens when I start the game and while at logo screen, I inject the DLL:

![image](https://github.com/Kyle873/SSR-KMod/assets/148178/87886ad0-60b0-45f8-9bc0-9b85a2e473d6)

Cheers,
Sun

**Comment by sunbeam906** [comment]
Looks like we'll have to wait for UE4SS 2.6, which has this issue fixed. I also tested it like this:

```
RegisterKeyBind(Key.DEL, function()
    local Hacker = FindFirstOf("CON_Hacker_C")
    if not Hacker:IsValid() then print("No Hacker found\n") return end
    Hacker:RefillVitals()
end)
```

..and it would still error :)

BR,
Sun

**Comment by Kyle873** [comment]
Yeah I need to do a lot more error checking. I've added a bunch, including fixing my bad checks for the regen stuff which was also probably throwing errors at game start as well if it was active.
